tests/contract_calls.py

```diff
@@ -120,13 +145,23 @@ class ReceiptAggregateVoucher(EIP712Message):
             print("Skip, signer already authorized")
         else:
             print(error)
+    check_subgraph_signer(SIGNER, True)
```

Comment:
Can you also add a test where a signer gets revoked?

Reply:
Okay